### PR TITLE
refactor: remove static jenkins

### LINKED ISSUE #7236: Remove jenkins chart
Static jenkins is no longer supported as a pipeline execution engine or webhook. We should remove the chart as well.

### PR DESCRIPTION
**Pull request by ankitm123**
fixes #7236

### CODE REVIEW

**Comment by abayer** [comment]
Ah, figured out why this failed - `jx/scripts/test.sh` is expecting to find images in `values.yaml`, and in fact, we actually still do need those pod templates to be around to figure out when to mount the Maven settings secret into a container, so let's bring that section (`jenkins.Agent.PodTemplates`) back, though I think we can get rid of the rest of what you deleted. We do need to eventually replace the pod templates with something else that can get us the mounts we need as well, but for now, it's easiest to just keep them here.

File: jenkins-x-platform/templates/jenkins-x-pod-templates.yaml [hunk 1]
```diff
@@ -59,84 +59,3 @@
   {{- end }}
   {{- end }}
 {{- end }}
-
-{{- if .Values.jenkins.Agent.Enabled }}
```
Comment (jstrachan) [inline jenkins-x-platform/templates/jenkins-x-pod-templates.yaml:63]: we still use the pod template ConfigMaps even when using Tekton

**Pull request closed without merging**